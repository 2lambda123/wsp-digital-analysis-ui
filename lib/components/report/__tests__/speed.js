/* global describe, it, expect */

import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'
import React from 'react'

import Speed from '../speed'

describe('Report > Speed', () => {
  it('renders correctly', () => {
    const props = {
      kmh: 2000
    }

    // mount component
    const tree = mount(
      <Speed
        {...props}
        />
      , {
        attachTo: document.getElementById('test')
      }
    )
    expect(mountToJson(tree)).toMatchSnapshot()
  })
})
