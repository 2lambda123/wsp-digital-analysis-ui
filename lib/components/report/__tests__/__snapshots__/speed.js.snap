exports[`Report > Speed renders correctly 1`] = `
<span>
  2000 km/h (1243 mph)
</span>
`;
